includes/admin/class-wp-job-manager-settings.php

```diff
@@ -335,6 +335,16 @@ protected function init_settings() {
 							'type'       => 'checkbox',
 							'attributes' => [],
 						],
+						[
+
+							'name'       => 'job_manager_enable_scheduled_listings',
+							'std'        => '0',
+							'label'      => __( 'Scheduled listings', 'wp-job-manager' ),
+							'cb_label'   => __( 'Enable Scheduled listings', 'wp-job-manager' ),
```

Comment:
Just letter casing to match the other settings:
```suggestion
							'label'      => __( 'Scheduled Listings', 'wp-job-manager' ),
							'cb_label'   => __( 'Enable scheduled listings', 'wp-job-manager' ),
```